ClearentIdtechIOSFramework/ClearentUI/ClearentProcessingModalViewController.swift

```diff
@@ -164,6 +166,12 @@ extension ClearentProcessingModalViewController: ClearentProcessingModalView {
             return signatureView()
         case .manualEntry:
             return manualEntryFormView()
+        case .error:
+            let errorDetailsView = ClearentErrorDetailsView()
+            guard let detailedErrorMessage = object as? String else { return nil }
+            
+            errorDetailsView.setup(with: detailedErrorMessage)
+            return errorDetailsView
```

Comment:
if you add `detailedErrorMessage` to `init`, this code simplifies also:
```suggestion
            return ClearentErrorDetailsView(detailedErrorMessage: detailedErrorMessage)
```